hare/hare.go

```diff
@@ -0,0 +1,177 @@
+package hare
+
+import (
+	"fmt"
+	"github.com/spacemeshos/go-spacemesh/common"
+	"github.com/spacemeshos/go-spacemesh/crypto"
+	"github.com/spacemeshos/go-spacemesh/hare/config"
+	"github.com/spacemeshos/go-spacemesh/mesh"
+	"sync"
+	"time"
+)
+
+const Delta = time.Second // todo: add to config
```

Comment:
Not really sure. this should be configured while we do testing. since now propagation delay doesn't matter I didn't add it or gave it a real value.

Reply:
currently `Hare` can be configured with a `networkDelta` time. but we should be able to load this setting from somewhere rather than using a constant.